src/scss/components/checkbox/_index.scss

```diff
@@ -142,120 +127,59 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
-      color: @cyan-900;
+      color: $cyan-900;
 
       ~ label > hx-checkbox {
-        border-color: @cyan-900;
-        color: @cyan-900;
+        border-color: $cyan-900;
+        color: $cyan-900;
       }
     }
 
     &:valid {
       ~ label > hx-checkbox:hover {
-        background-color: @cyan-50;
-        border-color: @cyan-500;
-        color: @cyan-500;
+        background-color: $cyan-50;
+        border-color: $cyan-500;
+        color: $cyan-500;
```

Comment:
This should be handled by the `hxCheckboxFacade` mixin.

Reply:
Replaced:
```scss
background-color: $cyan-50;
border-color: $cyan-500;
color: $cyan-500;
```

with:
```scss
@include hxCheckboxFacade($pseudo-state: hover);
```